### DISCUSSION TITLE
Is there any plan to make webserver as a plugin for airflow?

### DISCUSSION THREAD

**Discussion by broholens** [apache/airflow#22066] [Ideas] [answered]
The most vulnerabilities of airflow are about webserver, but some people  just use scheduler and they must fix the webserver vulnerabilities for security.

**Comment by kbumsik-angelswing** [comment]
I guess we can just run only the scheduler even for now, if it's the case?

**Reply by broholens** [reply]
Yes, you are right. But the vulnerabilities files still in the product environment, so it is still considered risky in some case.
I wish to install airflow by:
```
pip install airflow
pip install airflow-webserver
```

**Comment by potiuk** [comment, marked as answer]
No plans for now. But some of the changes we are implementing as part of AIP-1 https://cwiki.apache.org/confluence/display/AIRFLOW/AIP-1%3A+Improve+Airflow+Security and future plans abot splting airflow into separate repositories, might result in something like that eventually. This is how we `think` about airflow now that "Scheduler" is what really makes Airflow "core". The webserver on the other hand is thought as "add-on" indeed.

Watch the devlist for discussions about that in the coming months (and even if we will do something about that, this will take a year or so to implement so do not count on it any time soon.
